pkg/accesscontrol/access_store.go

```diff
@@ -31,10 +44,10 @@ type roleKey struct {
 }
 
 func NewAccessStore(ctx context.Context, cacheResults bool, rbac v1.Interface) *AccessStore {
```

Comment:
I know this part has already been removed, but it looks like `cacheResults` is always true in non-test calls to `NewAccessStore` , and there are no external non-test invocations of `AccessStore{...}`.   So why don't we assume caching is always on?

Reply:
I made myself the same question 😄
However, my intention was to keep the signature of exported methods unchanged.
Please consider opening a new PR with that proposal.